web/src/main/webapp/WEB-INF/pages/application_state.jsp

```diff
@@ -12,6 +12,8 @@
         <script src="<c:url value="/resources/js/atsy-statehistory-create.js" />"></script>
         <script src="<c:url value="/resources/thirdparty/bootstrap-validator/validator.js" />"
                         type="text/javascript"></script>
+        <script src="<c:url value="/resources/thirdparty/bootstrap-datepicker/bootstrap-datepicker.js" />" type="text/javascript"></script>
```

Comment:
Do not use jsp tags in attributes of other tag. Extract this `<c:out>` tag before the `<script>` tag.
Do this change for the changes you made, there is a separate task to fix this issue in the master branch (which is currently in progress).